vllm/executor/ray_gpu_executor.py

```diff
@@ -60,6 +63,15 @@ def _init_executor(self) -> None:
         # Create the parallel GPU workers.
         self._init_workers_ray(placement_group)
 
+        def enc_hook(obj: Any) -> Any:
+            if isinstance(obj, array):
+                # convert the complex to a tuple of real, imag
+                return obj.tobytes()
```

Comment:
Do we need else? If not, then should we use `assert`?

Reply:
ah good point. it looks like all unsupported types are coming here by default. So let me raise an exception

Reply:
Is the exception added somewhere else?

Reply:
oops. it was not pushed lol